lib/tty/prompt/list.rb

```diff
@@ -83,6 +83,21 @@ def default(*default_values)
         @default = default_values
       end
 
+      # Set confirm keys
+      #
+      # @param [Array<Symbol, String, Hash{Symbol, String => String}>] value
+      #   the new confirm_keys
+      #
+      # @return [Hash{Symbol, String => String}]
+      #
+      # @api public
+      def confirm_keys(value = (not_set = true))
+        return @confirm_keys if !@confirm_keys.nil? && not_set
+
+        value = not_set ? self.class::DEFAULT_CONFIRM_KEYS : value
+        @confirm_keys = init_action_keys(value)
+      end
```

Comment:
Looking at the examples made me want to be able to 'drop' array symbols and provide arguments directly:

```ruby
menu.confirm_keys :ctrl_s
menu.select_keys :enter
```

Given that `confirm_keys` isn't used internally we can avoid checking if `@confirm_keys` is nil and do the following:

```ruby
def confirm_keys(*keys)
  keys = keys.flatten
  return @confirm_keys if keys.empty?

  @confirm_keys = init_action_keys(keys)
end
```
This should give us more parameter flexibility. What do you think?